src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,169 @@
+# Pathfinder Method for Approximate Bayesian Inference {#pathfinder-config}
+
+The Pathfinder algorithm is described in section [Pathfinder overview](#pathfinder-intro).
+
+The full set of configuration options available for the `pathfinder` method is
+reported at the beginning of the pathfinder output file as CSV comments.
+When the example model `bernoulli.stan` is run with `method=pathfinder`
+via the command line with all default arguments,
+the resulting Stan CSV file header comments show the complete set
+of default configuration options:
+```
+method = pathfinder
+  pathfinder
+    init_alpha = 0.001 (Default)
+    tol_obj = 9.9999999999999998e-13 (Default)
+    tol_rel_obj = 10000 (Default)
+    tol_grad = 1e-08 (Default)
+    tol_rel_grad = 10000000 (Default)
+    tol_param = 1e-08 (Default)
+    history_size = 5 (Default)
+    num_psis_draws = 1000 (Default)
+    num_paths = 4 (Default)
+    save_single_paths = 0 (Default)
+    max_lbfgs_iters = 1000 (Default)
+    num_draws = 1000 (Default)
+    num_elbo_draws = 25 (Default)
+```
+
+The Pathfinder algorithm uses arguments `init_alpha` through `history_size` to run the L-BFGS optimizer.
+The remaining arguments are specific to the Pathfinder algorithm.
+
+## Pathfinder Configuration
+
+- `num_psis_draws` - Number of draws from PSIS sample (multi-path pathfinder).
+Must be a positive integer.  Default value is $1000$.
+
+- `num_paths` - Number of single pathfinders.
+Must be a positive integer.  Default value is $4$.
+
+- `save_single_paths` - When True ($1$), save outputs from single pathfinders.
+Valid values: $(0, 1)$. Default is $0$ (False).
+
+- `max_lbfgs_iters` - Maximum number of L-BFGS iterations.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_draws` - Number of approximate posterior draws for each single pathfinder.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_elbo_draws` - Number of Monte Carlo draws to evaluate ELBO.
+Must be a positive integer.  Default value is $25$.
+
+
+
+## L-BFGS Configuration
+
+These arguments have the same defaults as for [optimization](#optimize_config).
```

Comment:
You should also mention it's the same arguments and the same defaults as the args aren't listed anywhere I see.  It might help to call these out even if you don't repeat the doc.

Reply:
reworded to clarify.